src/commands/build.ts

```diff
@@ -85,30 +84,35 @@ export class BuildCommand implements Command {
 
   async run(options: CommandOptions, config: ProjectConfig): Promise<any> {
     // Defer dependency loading until this specific command is run
-    let build = require('../build/build').build;
-    const hasCLIArgumentsPassed = this.args.some(arg => {
-      return typeof options[arg.name] !== 'undefined';
-    });
+    let {build} = require('../build/build');
+    const {mainBuildDirectoryName} = require('../build/build');
```

Comment:
rather than `require()` twice, I think just do:

```
const buildLib = require(...);
let build = buildLib.build;
const mainBuildDirectoryName = buildLib.mainBuildDirectoryName;
```

If we could use destructuring, you could get some benefit:

```
const buildLib = require(...);
let {build} = buildLib;
const {mainBuildDirectoryName} = buildLib;
```